### ISSUE TITLE
Strong button affordance, or add a cursor pointer

### ISSUE THREAD

**Issue by camdenb** [kmikiy/SpotMenu#12]
This is a minor issue, but the Next/Previous track buttons should either be more clearly buttons, or the cursor should change into a pointer when hovering over them.

See: https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b#.s47d20e3t

**Comment by kmikiy** [comment]
That's been bothering me as well, just haven't had the time to fix it. I'll get into it later.

**Comment by camdenb** [comment]
@kmikiy Thanks! I can take a crack at it after work today -- which option are you leaning towards (better affordance vs cursor pointer)?

**Comment by kmikiy** [comment]
With the latest version of SpotMenu (1.3) almost the whole surface is clickable, so just simply using a cursor pointer might not help that much. Better affordance might be a better option, although I don't have any ideas on how to make the buttons look more like buttons and keep a minimal design at the same time.

**Comment by camdenb** [comment]
I disagree re: not using a cursor pointer, I think that adding one and maybe changing the arrow to a light grey on hover would help significantly.

I completely agree re: better affordance, though. That's a tough problem, especially if you're trying to be consistent with macOS styling. Which is why I think that the cursor pointer is better than nothing. :)

**Comment by TrevorSayre** [comment]
@camdenb I'm confused, the article you linked strongly disagrees with using `cursor: pointer` over buttons like the prev/next affordances in the menulet. And yet you seem to be advocating for their use. I think a hover state for the affordances would be sufficient.

**Comment by camdenb** [comment]
@trevorsayre Yeah, a hover state or better button affordance would be ideal, but as a bare minimum a pointer is better than *nothing*.

**Comment by jacobpgn** [comment]
👋 Hey, thanks for the project @kmikiy!

I was playing with this today and wanted to suggest a few small changes that might help here. Let me know if you agree, and I'll PR it!
- button alpha change on hover state
- more recognisable next/previous icons (hopefully without spoiling the clean/minimal look)
- smaller target areas to make it harder to hit the wrong button

e.g.
![kapture 2018-04-07 at 20 28 12](https://user-images.githubusercontent.com/6728978/38459495-398e3b5e-3aa2-11e8-96f1-b756440c268b.gif)

**Comment by kmikiy** [comment]
@jacobpgn nice, looks great🙌🏻 it would be awesome if you made a PR!

**Comment by camden** [comment]
Yeah, looks amazing!! Maybe the next/previous buttons could have some extra clickable space around them to make them a little easier to click?
